README.md

```diff
@@ -77,16 +77,30 @@ Here's the basic idea behind `nerves_time`:
   time known to `nerves_time`. This is either set in the application config or
   defaulted to a reasonable value that likely moves forward a little each
   `nerves_time` release.
-* Check for `~/.nerves_time`. If it exists, advance the clock to it's last
-  modification time.
+* check for time via a [Real Time Clock](#Real-Time-Clock)
 * Run Busybox `ntpd` to synchronize time using the [NTP
   protocol](https://en.wikipedia.org/wiki/Network_Time_Protocol).
-* Update `~/.nerves_time` periodically and on graceful power downs. This is
-  currently only done at around 11 minute intervals to avoid needless exercising
-  of Flash-based memory.
+* Update [Real Time Clock](#Real-Time-Clock) periodically and on graceful power 
+  downs. This is currently only done at around 11 minute intervals to avoid 
```

Comment:
I think that the "to avoid needless exercising of Flash-based memory" part can be deleted now.